apps/shared/avifutil.h

```diff
@@ -10,6 +10,55 @@
 extern "C" {
 #endif
 
+#define NEXTARG()                                                     \
+    if (((argIndex + 1) == argc) || (argv[argIndex + 1][0] == '-')) { \
+        fprintf(stderr, "%s requires an argument.", arg);             \
+        goto cleanup;                                                 \
+    }                                                                 \
+    arg = argv[++argIndex]
```

Comment:
Optional: I think it would be easier to understand (at the cost of code duplication) if this macro is defined in avifdec.c and avifenc.c as before. The reason is that this macro uses the local variable `argIndex`.

I am just bring this up for your consideration. It is fine to leave it here.

Reply:
Done